indexing-service/pom.xml

```diff
@@ -62,7 +62,11 @@
             <artifactId>druid-hll</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
-
+        <dependency>
```

Comment:
what is this used for ? lets move this to parent pom as done for other dependencies and just declare it here.

Reply:
Used for ` private final CircularFifoQueue<Long> lagMetricsQueue;` to collect lag points. Moved this to parent pom.